packaging/rhc-worker-bash.spec

```diff
@@ -27,7 +27,11 @@ URL:            https://github.com/%{repo_orgname}/%{repo_name}
 Source0:        %{url}/archive/v%{version}/%{name}-%{version}.tar.gz
 ExclusiveArch:  %{go_arches}
 
+%if 0%{?rhel} == 7
+BuildRequires:  go-toolset-1.16-golang
+%else
```

Comment:
Is there another condition we could use to avoid trying to install this package during the COPR build? We would like to still have those builds as part of our development setup, or in case anyone needs to do a quick testing, not a high priority for now, but if we can add another condition to skip the package in COPR, would be nice.

Reply:
I think we can limit the usage of that go toolset to RHEL only, and use the standard golang available otherwise in the distro. I made this change, and the copr build works again (also because the `epel-7-x86_64` target is CentOS+EPEL, whereas the `epel-8-*` targets are RHEL+EPEL; we'll fix that later on, in case).

Reply:
Even better: we should be able to use the standard golang also in RHEL 8+, so there should be nothing more to do, hopefully.